chain/client/src/stateless_validation/mod.rs

```diff
@@ -3,3 +3,5 @@ pub mod chunk_validator;
 pub mod processing_tracker;
 mod shadow_validate;
 mod state_witness_producer;
+
```

Comment:
```suggestion
```

Reply:
See https://github.com/near/nearcore/blob/master/docs/practices/style.md#import-blocks.  We should avoid unnecessary empty lines.